src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -356,21 +360,21 @@ private Color(long value, short state, string name, KnownColor knownColor)
             this.knownColor = unchecked((short)knownColor);
         }
 
-        public byte R => (byte)((Value >> ARGBRedShift) & 0xFF);
+        public byte R => (byte)(((uint)Value & ARGBRedMask) >> ARGBRedShift);
```

Comment:
Masking can be removed entirely from here - casting to byte takes care of that already.

Reply:
Oops, had my consistency blinders on ;)

Reply:
You can always send a follow up PR to fix this :)